velox/exec/tests/AggregationTest.cpp

```diff
@@ -444,7 +444,9 @@ TEST_F(AggregationTest, missingFunctionOrSignature) {
           const std::vector<TypePtr>& argTypes,
           const TypePtr& resultType,
           const core::QueryConfig& /*config*/)
-          -> std::unique_ptr<exec::Aggregate> { VELOX_UNREACHABLE(); });
+          -> std::unique_ptr<exec::Aggregate> { VELOX_UNREACHABLE(); },
+      true /*registerCompanionFunctions*/,
```

Comment:
This unit test doesn't use the companion functions. I'd suggest setting registerCompanionFunctions to false.